app/models/types/languages.py

```diff
@@ -0,0 +1,189 @@
+from enum import Enum, unique
+
+
+@unique
+class Languages(str, Enum):
```

Comment:
Can we add a comment stating which ISO standard this list originates from. Just so we can easily update it if/ when the standard changes.

Reply:
Added: `List of ISO 639-1 language codes.`